msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -361,6 +371,7 @@ void mergeConfiguration(PublicClientApplicationConfiguration config) {
         // Multiple is the default mode.
         this.mAccountMode = config.mAccountMode != AccountMode.MULTIPLE ? config.mAccountMode : this.mAccountMode;
         this.mClientCapabilities = config.mClientCapabilities == null ? this.mClientCapabilities : config.mClientCapabilities;
+        this.mIsSharedDevice = config.mIsSharedDevice == true ? this.mIsSharedDevice : config.mIsSharedDevice;
```

Comment:
What is the expected value of `this.mIsSharedDevice` here? Will it be `true` or can it also be `false`?

Reply:
the default value is false.